client/my-sites/plans/jetpack-plans/product-store/more-info-link/index.tsx

```diff
@@ -8,15 +8,20 @@ import './style.scss';
 export const MoreInfoLink: React.FC< MoreInfoLinkProps > = ( { item, onClick } ) => {
 	const translate = useTranslate();
 
-	const isExternalProduct = EXTERNAL_PRODUCTS_LIST.includes( item.productSlug );
+	const isExternalProductLink =
+		EXTERNAL_PRODUCTS_LIST.includes( item.productSlug ) && item.externalUrl;
 
-	const href = isExternalProduct && item.externalUrl ? item.externalUrl : `#${ item.productSlug }`;
+	const href = isExternalProductLink ? item.externalUrl : `#${ item.productSlug }`;
+
+	const target = isExternalProductLink ? '_blank' : undefined;
 
 	return (
-		<Button className="more-info-link" onClick={ onClick } href={ href } plain>
+		<Button className="more-info-link" onClick={ onClick } href={ href } target={ target } plain>
 			{ translate( 'More about {{productName/}}', {
 				components: { productName: <>{ item.shortName }</> },
 			} ) }
+
+			{ isExternalProductLink && <Gridicon icon="external" size={ 16 } /> }
```

Comment:
It would be good to have a prop to control whether this icon should be shown or not.

Reply:
Hi @manzoorwanijk, I decided to use isExternal props to determine if we are trying to render an external link.

Reply:
I didn't mean that. I meant to not always render that icon, but rather allow the consumer component to control whether `<Gridicon icon="external" />` should be displayed or not.

Reply:
Hi @manzoorwanijk,  I think this icon doesn't really represent anything other than an indicator that the rendered link is external URL. rather than making a standalone prop for controlling the icon. It would make our intention more clearer to distinguish an external to a non-external link.

Reply:
Alight, sounds good.